### ISSUE TITLE
Issue Transferring Funds to coinbase from gdax

### ISSUE THREAD

**Issue by rpringst** [danpaquin/gdax-python#176]
Wonderful library you have here, but I did find one glitch (And am not github savvy enough to upload the fix myself). When using the function coinbase_withdraw as part of the authenticated_client class, you always get a 'route not found' error. At line 254 of this file, it reads:

r = requests.post(self.url + "/withdrawals/coinbase", data=json.dumps(payload), auth=self.auth, timeout=30)

But the correct url, as per my scouring of the GDAX API, is actually found at "/withdrawals/coinbase-account". Thanks in advance, brev!

**Comment by derenma** [comment]
Is it possible that you are confusing 'withdraw'  with 'coinbase_withdraw' or 'crypto_withdraw'?  Submitting a PR that may fix.

**Comment by rpringst** [comment]
No, the issue is that the rest api url for that method is incorrect.
Compare and contrast each pair of methods, and you'll see that. I already
fixed it on the library installed on my system, works perfectly, I just
don't 'github' much...

**Comment by danpaquin** [comment]
This remains an issue that needs to be updated ASAP in addition to supporting all payment-types.

If this issue isn't closed, anyone capable of submitting a PR for this would be appreciated.
